integration/test/upgradetiller/upgrade_tiller_test.go

```diff
@@ -19,55 +19,64 @@ func TestUpgradeTiller(t *testing.T) {
 
 	var err error
 
-	labelSelector := "app=helm,name=tiller"
+	currentTillerImage := "quay.io/giantswarm/tiller:v2.8.2"
 	outdatedTillerImage := "gcr.io/kubernetes-helm:v2.7.2"
+
+	labelSelector := "app=helm,name=tiller"
 	tillerNamespace := "giantswarm"
 
```

Comment:
Below `currentHelmClient` and `outdatedHelmClient` are set and used in the test.